### ISSUE TITLE
/kill Bot#1 will make you suicide

### ISSUE THREAD

**Issue by necessarily-equal** [Unvanquished/Unvanquished#1279]
`/kill` shouldn't cause a suicide if you don't give any argument.

I'll have a look

**Comment by Pretuer** [comment]
Will test this

**Comment by necessarily-equal** [comment]
The code is in src/sgame/sg_cmds.cpp, around line 780. It seems it 
indeed takes no argument.

**Comment by Pretuer** [comment]
yes it works, I think that a /killme command should be made but /kill is okay. I think it's best to ask the community on this

https://user-images.githubusercontent.com/39959123/104089235-47ddb080-527e-11eb-9371-567cb1a794cd.mp4

**Comment by Pretuer** [comment]
https://git.io/JLhkJ

**Comment by illwieckz** [comment]
`kill` is indeed the suicide command. Its name is inherited from Quake3 as far as I know (so, before year 2000)…

**Comment by necessarily-equal** [comment]
that command is older than me 😅️.

How about having `/kill` to kill the matching arguments (like `/kick`), 
and /killme to suicide? (I can make the change)

**Comment by bmorel** [comment]
To avoid breaking old confs (sigh) I think `/kill` without arg is ok for suicide, but supporting arguments would be nice indeed.
Likely, giving orders to suicide to bots, or for admins to be able to kill a "bad person" or even buildables placed in trolling places.
This would still have problems though, as killing friendly bots is a good way to troll own team. I'm very mitigated to see that kind of features in the hands of non-registered players.
